SIGs/SIG-embedded/proposal.md

```diff
@@ -0,0 +1,36 @@
+# Proposal to Create a SIG-Embedded and Industrial Applications
+
+This document is a proposal to create SIG-Embedded and Industrial Applications as a formal Special Interest Group (SIG) under the auspices of the TSC of the Bytecode Alliance, as specified in the TSC’s charter. 
+
+The Embedded ecosystem is ideally placed to benefit from the use of WASM and WASI. It has a set of constraints on performance in comparison to native non—WASM execution, determinism and reliability which can exceed those of other ecosystems. This gives rise to the definition of a low footprint device. The best way to address these specific needs is with a unified ecosystem of tools, technologies, documentation, and support. 
+
+The purpose of this group is to support and promote the use of WebAssembly and WASI in Embedded and Industrial user cases. This will be achieved by providing notes on best practice to existing practitioners, and by providing the TSC, WASI subgroup and WASM community group with uses cases which illustrate constraints and proposals to address such limitations.
+
+Any work the group does which involves standardized extensions to runtimes and their interfaces will enter the appropriate standardization process (e.g. WASM CG, WASI).
+
+To ensure that all interested members of the WebAssembly community can participate the SIG will arrange regular meetings which rotate between time slots most suitable for Asia, Europe and the Americas. It would therefore be desirable to have three co-chairs, preferably, one from each time-zone who would help facilitate the regular meetings.
+
+## Deliverables
+
+1. In order to provide scope and guide discussions the SIG will help provide a definition of the lowest addressable device (low footprint device) that WASM and WASI could operate on.
```

Comment:
```suggestion
1. In order to scope and guide discussions, the SIG will define the lowest addressable device (low footprint device) that Wasm and WASI can target.
```

Reply:
Thanks Nick, this looks cleaner and achieves the same ends.